modules/api/pkg/handler/v2/routes_v2.go

```diff
@@ -1512,6 +1513,119 @@ func (r Routing) RegisterV2(mux *mux.Router, oidcKubeConfEndpoint bool, oidcCfg
 		Handler(r.getSeedOverview())
 }
 
+// swagger:route GET /api/v2/projects/{project_id}/disktypes project listProjectGCPDisk
+//
+//	List disktypes for a given project
+//
+//	Consumes:
+//	- application/json
+//
+//	Produces:
+//	- application/json
+//
+//	Responses:
+//	  default: errorResponse
+//	  200: GCPDiskTypeList
+func (r Routing) listProjectGCPDiskTypes() http.Handler {
+	return httptransport.NewServer(
+		endpoint.Chain(
+			middleware.TokenVerifier(r.tokenVerifiers, r.userProvider),
+			middleware.UserSaver(r.userProvider),
+		)(provider.ListProjectGCPDiskTypes(r.presetProvider, r.userInfoGetter)),
+		provider.DecodeProjectGCPDisktypes,
+		handler.EncodeJSON,
+		r.defaultServerOptions()...,
+	)
+}
+
+// swagger:route GET /api/v2/projects/{project_id}/providers/gcp/subnetworks gcp listProjectGCPSubnetworks
+//
+// Lists available GCP subnetworks
+//
+//	Produces:
+//	- application/json
+//
+//	Responses:
+//	  default: errorResponse
+//	  200: GCPSubnetworkList
+func (r Routing) listProjectGCPSubnetworks() http.Handler {
+	return httptransport.NewServer(
+		endpoint.Chain(
+			middleware.TokenVerifier(r.tokenVerifiers, r.userProvider),
+			middleware.UserSaver(r.userProvider),
+		)(provider.ListProjectGCPSubnetworks(r.presetProvider, r.userInfoGetter, r.seedsGetter)),
+		provider.DecodeProjectGCPSubnetworks,
+		handler.EncodeJSON,
+		r.defaultServerOptions()...,
+	)
+}
+
+// swagger:route GET /api/v2/projects/{project_id}/clusters/{cluster_id}/providers/gcp/networks gcp listProjectGCPNetworks
```

Comment:
Should be:

```suggestion
// swagger:route GET /api/v2/projects/{project_id}/providers/gcp/networks gcp listProjectGCPNetworks
```

(without the cluster content).